app/models/work_state_transition_notification.rb

```diff
@@ -13,7 +13,7 @@ def initialize(work, current_user_id)
     @collection_administrators = work.collection.administrators.to_a
     @work_url = Rails.application.routes.url_helpers.work_url(work)
     @work_title = work.title
-    @users = @collection_administrators + [@depositor]
+    @users = @collection_administrators | [@depositor] # Depositor may also be an admin, but should only be listed once.
```

Comment:
This is pretty neat, I didn't know you could use `|` like this to prevent duplicates.